addons/website_forum/models/res_users.py

```diff
@@ -32,57 +32,6 @@ def _get_user_waiting_post(self):
             domain = [('parent_id', '=', False), ('state', '=', 'pending'), ('create_uid', '=', user.id)]
             user.forum_waiting_posts_count = Post.search_count(domain)
 
-    @api.model
-    def _generate_forum_token(self, user_id, email):
-        """Return a token for email validation. This token is valid for the day
-        and is a hash based on a (secret) uuid generated by the forum module,
-        the user_id, the email and currently the day (to be updated if necessary). """
-        forum_uuid = self.env['ir.config_parameter'].sudo().get_param('website_forum.uuid')
```

Comment:
in the `init` method of forum.forum, this parameter is created. If it's not used anywhere else, you should remove that code.
And create the same for your new `website_profile.uuid` (on res.users).

Reply:
As discussed, we can directly integrate the profile_uuid inside the generate_profile_token to set the param at first call. as it's only used here.. Baboum !